precompile/contracts/nativeminter/config.go

```diff
@@ -7,32 +7,23 @@ import (
 	"fmt"
 	"math/big"
 
-	"github.com/ava-labs/subnet-evm/precompile"
 	"github.com/ava-labs/subnet-evm/precompile/allowlist"
+	"github.com/ava-labs/subnet-evm/precompile/config"
 	"github.com/ava-labs/subnet-evm/utils"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/math"
 )
 
-var _ precompile.StatefulPrecompileConfig = &ContractNativeMinterConfig{}
-
-// ConfigKey is the key used in json config files to specify this precompile config.
-// must be unique across all precompiles.
-const ConfigKey = "contractNativeMinterConfig"
+var _ config.Config = &ContractNativeMinterConfig{}
```

Comment:
wonder if we could call this `Config` so externally it would be referenced as `nativeminter.Config{}`